### ISSUE TITLE
咨询一下关于mobilenet的训练结果

### ISSUE THREAD

**Issue by zjchuyp** [Sanster/tf_ctpn#3]
作者您好，您mobilenet v2的模型训练成功了吗？我用您的程序和您提供的数据训练mobilenet，好像没有成功收敛。测试结果也比vgg16要差很多，而且mobilenet v2模型的前向速度比vgg16的模型好像也没快多少，不知是哪儿弄错了。

**Comment by zjchuyp** [comment]
![qq 20180827141729](https://user-images.githubusercontent.com/12995459/44643679-f5c8c700-aa03-11e8-8360-8f0d58714651.png)

**Comment by Sanster** [comment]
是用新的那个数据集吗？在 icdar13 上面跑出来多少？ mobilenet v2 我只用 text-detect-ctpn 的数据集练过，不过没在 icdar13 测过。速度上，我在移动端测下来是比 vgg16 快很多的，PC 端都用 gpu 跑的话速度可能是快不了多少

**Comment by zjchuyp** [comment]
是的，用的是您加了mlt17的数据集，没跑icdar13呢，我只测了十几张自己的证件图像，vgg16的结果很好，而mobilenet很多文字没检测出来。是不是训练了50k轮还不够，我训练训练看看效果。谢谢您回复啊。

**Comment by Sanster** [comment]
不客气。你有加载 ImageNet pretrain 模型来 fine turning 吗？ vgg16 我训练的时候是有的

**Comment by zjchuyp** [comment]
是finetune的方式的，是根据您readme里讲的train的方法，只是下载

MobileNet_v2_1.4_224^* |  
-- | --

这个模型, 把vgg16相关的改成这个来训练。

**Comment by Sanster** [comment]
原版的 mobilenet_v2 我记得 total stride 是 32，CTPN 要求是 16，所以我该了下 `V2_DEF` -> `CTPN_DEF`，使得 total stride 变成 16，不确定这会不会影响 pretrain model 的加载。

**Comment by zjchuyp** [comment]
我估计有影响

**Comment by Nic-Ma** [comment]
Hi @zjchuyp @Sanster 

请问现在训练成功MobileNet v2版本了吗？
我在Caffe上尝试训练MobileNet版本的CTPN。
Thanks.

**Comment by zjchuyp** [comment]
@toxic2m 现在没在搞ctpn了，用了其他的网络结构了

**Comment by Nic-Ma** [comment]
@zjchuyp 你后面也没搞定把卷积换成MobileNet v2？
请问你现在用的什么网络做文字检测？
Thanks.

**Comment by zhcheng26** [comment]
Hi   @toxic2m   你实现了在caff下的细节训练了吗？我实现过程中一直碰到了问题，想请教下？

**Comment by Nic-Ma** [comment]
Hi @zhcheng26 VGG的能跑训练了，MobileNet还在搞，我是基于https://github.com/zhouyoulie/ctpn_train

**Comment by zhcheng26** [comment]
Hi @toxic2m   正在做MobileNet v2的改善，基于的也是同一个demo训练，结果出来了，但是https://github.com/zhouyoulie/ctpn_train  他其中包含的xside 主要是针对长文本做出的改善吗？ 没有深入理解它改善的地方？能简单说说你的理解吗？

**Comment by zjchuyp** [comment]
@toxic2m yolo3

**Comment by Nic-Ma** [comment]
@zjchuyp yolo3不太适合做文本检测吧？你测过和CTPN对比准确度、速度吗？
Thanks.

**Comment by Nic-Ma** [comment]
@zhcheng26 Github比较少上，邮件沟通吧：nma@nvidia.com
CTPN作者田值博士让我不要用xside，注释掉了。
Thanks.

**Comment by zjchuyp** [comment]
@toxic2m 用yolo3替换ctpn中的faster-rcnn部分，后处理还是一样。效果很好，速度也飞快。

**Comment by Sanster** [comment]
@zjchuyp 用 yolo3 训练，训练集的 label 还是从大 box split 出来的小 box？

**Comment by kasyoukin** [comment]
在用vgg16在预测的时候，速度很不稳定，不知道大家是不是也一样，https://github.com/tensorflow/tensorflow/issues/22369

为了模型能块一点，我也练过resnet101,和 mobilenetv2，但效果都与VGG16有一定的差距，难以实用。

**Comment by edis219** [comment]
@zjchuyp yolo3 对于非常小的文字区域 识别怎么样？
